api/controllers/FlairController.js

```diff
@@ -168,8 +168,8 @@ module.exports = {
         }
       });
 
-      var newPFlair = _.get(req, "user.flair.ptrades.current.flair_css_class") || "default";
-      var newsvFlair = _.get(req, "user.flair.svex.current.flair_css_class") || "";
+      var newPFlair = _.get(req, "user.flair.ptrades.flair_css_class", "default");
```

Comment:
Also, wait, I checked this and the object doesn't contain flair_css_class, it contains current which contains flair_css_class.